pytest_indexd/hints.py

```diff
@@ -0,0 +1,14 @@
+try:
```

Comment:
I believe there was a `.pyi` file in this review. This module seems to be doing the same type of thing. Perhaps standardize on the extension name.